core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnMLOpDesc.scala

```diff
@@ -13,7 +13,7 @@ abstract class SklearnMLOpDesc extends PythonOperatorDescriptor {
   var model = ""
 
   @JsonIgnore
-  var name = ""
+  var modelName = ""
```

Comment:
modelName is confusing here. Developers might use the sklearn model name here when they build a new child class.
For example, they could use `KNeighborsClassifier` instead of the user-friendly name `K-nearest Neighbors`.